### ISSUE TITLE
Казахские символы

### ISSUE THREAD

**Issue by Dafafgh** [VitalyKravtsov2016/OposWebkassa#46]
В асу добавили поддержку казахских символов. Можно добавить поддержку казахских символов в OPOS?, сейчас печатаются знаки вопроса:
17:19:43.341  487.......  Info......  TItemTrans..........  ValidateOnPhase.....  ItemTrans Алмұртә ItemId=24275 (30021) Amount=420 Qty=1 Weight=1 BaseUnitPrice=420 UnitPrice=420 LineDiscount=0 Discount=0 TaxMask=4 AbsoluteIndex=1

[17.04.2023 17:19:41.554 ] [00001856] [DEBUG] ToleFiscalPrinter.PrintRecItem('Алм?рт?', 420, 1000, 4, 420, 'кг')
[17.04.2023 17:19:41.554 ] [00001856] [DEBUG] ToleFiscalPrinter.PrintRecItem('Алм?рт?', 420, 1000, 4, 420, 'кг')=0

[SHTRIH-M-OPOS-1_2023.04.17.log](https://github.com/VitalyKravtsov2016/OposWebkassa/files/11251356/SHTRIH-M-OPOS-1_2023.04.17.log)

**Comment by VitalyKravtsov2016** [comment]
Добрый день. 
Хотелось бы увидеть данные, которые передаются в драйвер.
Для этого можно использовать CCO Debug Runtime:
http://www.monroecs.com/files/OposCCOsDebug-1_14_001.zip
Лог файлы будут находиться в корне диска С:\

**Comment by Dafafgh** [comment]
Добрый день
не получается поставить дебаг версию. удалил боевую, зарегистрировал dll\ocx дебаговской версии вручную. логи не пишутся. нет ли инструкции по установке?

[RegComSvr.log](https://github.com/VitalyKravtsov2016/OposWebkassa/files/11259909/RegComSvr.log)

**Comment by VitalyKravtsov2016** [comment]
Добрый день. 
При регистрации нужны права администратора. 
Нужно запустить установку или регистрацию от имени администратора.

**Comment by Dafafgh** [comment]
да, нужно было запускать из учетной записи администратора, запуск от имени администратора не срабатывал
логи не пишутся, больше никаких настроек не нужно?

[RegComSvr.log](https://github.com/VitalyKravtsov2016/OposWebkassa/files/11260440/RegComSvr.log)

**Comment by VitalyKravtsov2016** [comment]
Логи должны писаться. 
Проверить можно при помощи теста OPOS драйвера ФР. 
У меня пишется в файл C:\20230418-OposCCO_FiscalPrinter.log

**Comment by Dafafgh** [comment]
лог писался в другую директорию

[18.04.2023 14:35:42.617 ] [00003488] [DEBUG] ToleFiscalPrinter.PrintRecItem('Алм?рт?', 4, 10, 4, 420, 'кг')
[18.04.2023 14:35:42.617 ] [00003488] [DEBUG] ToleFiscalPrinter.PrintRecItem('Алм?рт?', 4, 10, 4, 420, 'кг')=0


[20230418-OposCCO_FiscalPrinter.log](https://github.com/VitalyKravtsov2016/OposWebkassa/files/11261224/20230418-OposCCO_FiscalPrinter.log)

**Comment by VitalyKravtsov2016** [comment]
Сам принтер не поддерживает ни UNICODE, ни казахские символы. 
Просто драйвер принтера печатает эти сиволы в графике.

**Comment by VitalyKravtsov2016** [comment]
Добавлены:
https://github.com/VitalyKravtsov2016/OposWebkassa/releases/tag/0.63
